content/docs/setup/kubernetes/multicluster-install/gateways/index.md

```diff
@@ -5,8 +5,8 @@ weight: 2
 keywords: [kubernetes,multicluster,federation,gateway]
 ---
 
-Instructions for installing an Istio mesh across multiple clusters when pods
-in each cluster can only connect to remote gateway IPs.
+Instructions for installing an Istio [multicluster service mesh](/docs/concepts/multicluster-configuration/)
```

Comment:
"when pods ..." replace with 
`
where Kubernetes cluster services and applications in each cluster are limited to communication using the gateway IPs.
`